Entities/Items/Food/Eatable.as

```diff
@@ -10,6 +10,25 @@ void onInit(CBlob@ this)
 	this.addCommandID(heal_id);
 }
 
+void onTick(CBlob@ this)
```

Comment:
The whole hook seems unnecessary as you could get the username in the `onRender` hook instead, without having to use a separate property for the player name and network ID.

Reply:
should be fine now